tests/test_pkg_integrity.py

```diff
@@ -4,7 +4,8 @@
 import shutil
 import tempfile
 import unittest
-from unittest.mock import patch
+from unittest import mock
+from mock import patch
```

Comment:
Shouldn't we import from `unittest.mock`? `mock` is a backport package.

Reply:
Ah

Reply:
Didn't realize I was shadowing lol

Reply:
This passed on my system without issue and so I didn't think about it much.